packages/@aws-cdk/aws-apigatewayv2-alpha/lib/http/route.ts

```diff
@@ -193,22 +193,36 @@ export class HttpRoute extends Resource implements IHttpRoute {
       scope: this,
     });
 
-    this.authBindResult = props.authorizer?.bind({
-      route: this,
-      scope: this.httpApi instanceof Construct ? this.httpApi : this, // scope under the API if it's not imported
-    });
+    if (props.authorizer) {
+      this.authBindResult = props.authorizer.bind({
+        route: this,
+        scope: this.httpApi instanceof Construct ? this.httpApi : this, // scope under the API if it's not imported
+      });
+    } else if (this.httpApi instanceof HttpApi && this.httpApi.defaultAuthorizer) { // because IHttpApi as it is does not have a defaultAuthorizer
+      this.authBindResult = this.httpApi.defaultAuthorizer.bind({
+        route: this,
+        scope: this.httpApi, // this.httpApi is also a Construct because it is an HttpApi
+      });
+    }
```

Comment:
```suggestion
    const authorizer = props.authorizer ?? this.httpApi.defaultAuthorizer;
    this.authBindResult = authorizer?.bind({
      route: this,
      scope: this.httpApi instanceof Construct ? this.httpApi : this, // scope under the API if it's not imported
    });
```
It should be fine to add `defaultAuthorizer` and `defaultAuthorizationScopes` to the `IHttpApi` interface (unless I'm missing something).

Reply:
https://github.com/aws/aws-cdk/pull/27576/commits/ae2a40c7bb9cf7f91d40692bcd51dabe10e14ed6#diff-a58017864138a66a728474ed6d0cddd037026dc5fd39597b08b7be3dbc8d2150R25-R38

https://github.com/aws/aws-cdk/pull/27576/commits/ae2a40c7bb9cf7f91d40692bcd51dabe10e14ed6#diff-fc1d30aabf39df4c28614d540398d745d5d46aeac0c88dcdeda5c18e7cf596e9R196-R200